src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -283,12 +310,14 @@ object AddDatasetMetadataToDeposit {
     <key>{value}</key>.copy(label=key)
     // @formatter:on
   }
+}
 
+object validators {
   /**
    * Check if either non of the keys have values or all of them have values
    * If some are missing, mention them in the exception message
    */
-  def checkAllOrNone(row: Int, map: mutable.HashMap[MultiDepositKey, String], keys: List[String]): Try[Unit] = {
+  def checkAllOrNone(row: Int, map: mutable.HashMap[MultiDepositKey, String], keys: String*): Try[Unit] = {
```

Comment:
Maybe we can use a type for mutable.HashMap[MultiDepositKey, String],
 it's almost a 'Dataset'...?